src/core_plugins/kibana/public/dashboard/dashboard.html

```diff
@@ -1,6 +1,6 @@
 <dashboard-app class="app-container dashboard-container">
   <div class="fullScreenModePlaceholder">
-    <button
+    <div
```

Comment:
used `kbn-accessible-click`, that seemed to work better, otherwise I couldn't seem to press "enter" once it was highlighted.